docs/src/components/__tests__/Tabs.test.js

```diff
@@ -0,0 +1,49 @@
+import { fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import { Tabs, TabItem } from '../Tabs';
+
```

Comment:
In this case as there are things rendered, we. should be using `screen` instead of container